v-next/hardhat-errors/src/errors.ts

```diff
@@ -81,6 +81,9 @@ export class HardhatError<
       messageArgumentsOrParentError === undefined ||
       messageArgumentsOrParentError instanceof Error
     ) {
+      /* eslint-disable @typescript-eslint/consistent-type-assertions -- 
+      Typescript infercente get's lost here, but we know that if we didn't get
```

Comment:
typo: _infercente_, I don't know why it doesn't let me submit a code suggestion.

Reply:
Thanks!